lib/full-site-editing/template-parts.php

```diff
@@ -168,19 +168,36 @@ function set_unique_slug_on_create_template_part( $post_id ) {
  * @return array The supported template part area values.
  */
 function gutenberg_get_allowed_template_part_areas() {
```

Comment:
What happens to `WP_TEMPLATE_PART_AREA_SIDEBAR`? I know it's not been included to template part block variations. Is still valid but not exposed yet? Because it has code references like here: https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/navigation-sidebar/navigation-panel/constants.js#L94

Reply:
> What happens to WP_TEMPLATE_PART_AREA_SIDEBAR?

Basically getting rid of its definition here for now since it isn't yet supported in the editor.  Originally we were just defining and supporting header, footer, and general here - but I defined sidebar here knowing it would *eventually* be supported in the front-end, and thought there would be no harm in declaring it as a valid area on the back end in the meantime.  However, since we are refactoring these definitions to be in-sync with the front end (send them to settings, store, .map them as options in inputs, etc.) it is now more important that they are 1-1 with what the editor can handle.  Once we are ready to support sidebars in the editor, we would add their definition here again.